packages/aws_common/lib/src/config/aws_service.dart

```diff
@@ -0,0 +1,1007 @@
+import 'package:aws_common/aws_common.dart';
```

Comment:
Thanks for adding this! I hope this file is generated for 1000+ lines? 😅

nit: license header if this file is not generated...

Reply:
It's generated lol. I've added the license header.